source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -13,6 +13,7 @@ $font-family: 'Segoe UI', sans-serif;
 
             &:hover {
                 background-color: $selectable-hover-background !important;
+				cursor: pointer;
```

Comment:
Is there a difference between `hover { cursor: pointer}` or simply `{cursor: pointer}` ?
I don't think so and it's probably a better to have `{cursor: pointer}` for this class in general although it yields the same result :)

Reply:
just to jump on here with the spacing :)

@anna-dingler - if possible you should configure your editor to apply `prettier` formatting. If you're using VSCode, you can use the `Prettier` extension. More info on setting that up [here](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).

Reply:
Making that change now :)

I need to set my tabs to be spaces, and I'll look at the prettier formatting too